wrappers/core18_test.go

```diff
@@ -95,6 +105,7 @@ WantedBy=snapd.service
 		{"start", "usr-lib-snapd.mount"},
 		{"daemon-reload"},
 		{"--root", dirs.GlobalRootDir, "enable", "snapd.service"},
+		{"--root", dirs.GlobalRootDir, "enable", "snapd.system-shutdown.service"},
```

Comment:
Where is this coming from?

Reply:
this is used (or used to be used) for image creation, with services shipping enabled. I'm not sure if it's  still used.